public/index.html

```diff
@@ -3,15 +3,31 @@
 
 <head>
     <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Index</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+    <title>Cute Kiwi Inc.</title>
 </head>
 
 <body>
-    <a href="./login/login.html">Login</a>
-    <a href="./login/register.html">Register</a>
-    <a href="./content/">Content</a>
-    <a href="/api/logout">Logout</a>
+    <div class="container">
+        <div class="row">
+            <div class="col">
+                <h1 class="text-center">Cute Kiwi Inc.</h1>
+            </div>
+        </div>
+        <br>
+        <div class="row">
+            <div class="col"></div>
+            <div class="col">
+                <a href="./login/login.html" role="button" class="btn btn-dark btn-lg btn-block">Login</a>
+            </div>
+            <div class="col">
+                <a href="./login/register.html" role="button" class="btn btn-dark btn-lg btn-block">Register</a>
```

Comment:
There has to be button to a content because I can navigate to `/index.html` through url after login and there is no way to navigate back to the `/content`.

Reply:
Ideally, if you are logged in, then `/index.html` should redirect to content page, because it doesn't make sense to have an access to login/register page if you are already logged in.

Reply:
We are using a file server to deliver web pages. I am not going to modify its logic just to redirect to index.html. It is much easier to add a bunch of buttons.

Reply:
Done.